ArtOfIllusion/src/artofillusion/ViewerCanvas.java

```diff
@@ -470,7 +470,7 @@ public void preparePerspectiveAnimation()
   /**
     ViewAnimation calls this when the animation is finished, so the orientation menu
     will be up to date and the perspective is set to it's value without launched
```

Comment:
launched at this line and launching again on next

Reply:
I'll catch it in the rebase.